app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,35 @@
+<br>
+<br>
+<div class ="container-fluid">
+  <h2>Sign up</h2>
+  <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+    <%= render "devise/shared/error_messages", resource: resource %>
+    <div class="field">
+      <%= f.label :name %><br />
+      <%= f.text_field :name, autofocus: true, autocomplete: "name" %>
```

Comment:
- [ ] I get an error when I want to create an account and this is despite filling the name input as you can see in the attached image. So I could not confirm others such as the reset password fro the recoverable.
![image](https://github.com/ANTOINE1128/Blog-App1/assets/86200356/a2819093-9702-4357-8336-ba2e8bdb329d)